test/mitmproxy/test_examples.py

```diff
@@ -1,11 +1,33 @@
 import glob
+import os
+from contextlib import contextmanager
+
 from mitmproxy import utils, script
 from mitmproxy.proxy import config
-from . import tservers
+from netlib import tutils as netutils
+from netlib.http import Headers
+from . import tservers, tutils
+
+example_dir = utils.Data(__name__).path("../../examples")
+
+
+@contextmanager
+def example(command):
+    command = os.path.join(example_dir, command)
+    ctx = DummyContext()
```

Comment:
I think it's perfectly fine to mock these in the DummyContext. Should solve the issue above as well, shouldn't it?